CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * [#7007](https://github.com/rubocop-hq/rubocop/pull/7007): Fix `Style/BlockDelimiters` with `braces_for_chaining` style false positive, when chaining using safe navigation. ([@Darhazer][])
 * [#6880](https://github.com/rubocop-hq/rubocop/issues/6880): Fix `.rubocop` file parsing. ([@hoshinotsuyoshi][])
 
+### Changes
+
+* [#7027](https://github.com/rubocop-hq/rubocop/pull/7027): Allow unicode/display_width dependency version 1.6.0. ([@tagliala][])
```

Comment:
That's not a user visible change, so probably we shouldn't document it here.

Reply:
@bbatsov thanks for the feedback.

I've based this PR on #6813, so I've used the very same approach

I've added a line in the changelog because I've found similar entries (eg: https://github.com/rubocop-hq/rubocop/commit/cdf442bfa828d0ab5467c0291863893b17f28c28)

I have no problem in reverting the changes on Readme, let me know

Reply:
Not a big deal for me. Just rebase on top of `master` then and we're done.